ios/FluentUI/Tab Bar/TabBarTokenSet.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Design token set for the `TabBar`.
+public class TabBarTokenSet: ControlTokenSet<TabBarTokenSet.Tokens> {
+    public enum Tokens: TokenSetKey {
+        /// Defines the background color of the  of the `TabBarItem` when selected.
+        case tabBarItemSelectedColor
+
+        /// Defines the background color of the  of the `TabBarItem` when not selected.
+        case tabBarItemUnselectedColor
+
+        /// Font info for the title label when in portrait view
```

Comment:
Nit: period (I'm sorry for all the nits)

Reply:
Thanks for pointing all these out!